pyproject.toml

```diff
@@ -37,7 +37,7 @@ dependencies = [
     'panel >=1.0.0',
     'param >=1.9.3,<3.0',
     'pyproj',
-    'shapely',
+    'shapely >=1.8.0',
```

Comment:
Was released in October 25th, 2021

<img width="918" height="321" alt="image" src="https://github.com/user-attachments/assets/61751fdc-f499-447e-a2d9-0764b6d3c4b0" />